src/views/AddItem.jsx

```diff
@@ -116,17 +119,30 @@ export function AddItem({ listId, data }) {
 				</div>
 			</div>
 			<form onSubmit={handleSubmit}>
-				<label htmlFor="item">Item:</label>
-				<input
-					type="text"
-					id="item"
-					name="item"
-					placeholder="Enter Item"
-					value={itemName}
-					onChange={(e) => {
-						setItemName(e.target.value);
-					}}
-				/>
+				<div className="flex flex-col justify-center items-center">
+					<label htmlFor="item" className="px-3 mt-2">
+						Item:
+					</label>
+					<div className="w-full sm:w-1/2 flex items-center border-2 rounded-lg py-2 px-3">
+						<FontAwesomeIcon
+							icon={pen}
+							title="Enter item name"
+							className="text-gray-500 mr-2 sm:mr-4"
+						/>
+						<input
+							className="flex-grow border-none outline-none bg-transparent"
+							type="text"
+							id="item"
+							name="item"
+							placeholder="Enter Item"
```

Comment:
I like the gray, but our original design had the dark green for the placeholder in light mode and light blue in light mode. I'm fine either way, just thought I'd mention it jut in case.